### PR TITLE
Update man pages for product commands

### PR DESCRIPTION
**Pull request by KevinAnsleyJohnson**
Signed-off-by: Kevin Johnson <kevin_johnson@cargill.com>

### CODE REVIEW

**Review by dnewh — DISMISSED** [review]

**Review by vaporos — COMMENTED** [review]
See comments on #736, which would apply here as well.

**Review by vaporos — CHANGES_REQUESTED** [review]
Meant to request changes - See comments on #736, which would apply here as well.

File: cli/man/grid-product-list.1.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ NAME
...
 ========
 
-**grid product list** \[**FLAGS**\]
+**grid product list** \[**FLAGS**\] \[**OPTIONS**\]
 
 DESCRIPTION
 ===========
 
-List all products in grid. If the `service_id` flag is specified, only
+List all products in grid. If the `service_id` option is specified, only
```
Comment (KevinAnsleyJohnson) [inline cli/man/grid-product-list.1.md:21]: According to the implementation, we _still_ use GRID_SERVICE_ID env var to set the default. Is this what we want? or should the implementation change for `grid product list` to not use the GRID_SERVICE_ID env var?

This man-page is accurate to the implementation. Happy to take this on if needed, outside the current story.

File: cli/man/grid-product-list.1.md [hunk 2]
```diff
@@ -82,11 +82,10 @@ ENVIRONMENT VARIABLES
 =====================
 
 **`GRID_DAEMON_ENDPOINT`**
-: Specifies the endpoint for the grid daemon (`gridd`)
-  if `-U` or `--url` is not used.
+: Specifies a default value for `--url`
 
 **`GRID_SERVICE_ID`**
-: Specifies service ID if `--service-id` is not used
+: Specifies a default value for `--service-id`
```
Comment (KevinAnsleyJohnson) [inline cli/man/grid-product-list.1.md:88]: Yes - it's used as a default here. maybe that is acceptable?

**Review by dplumb94 — CHANGES_REQUESTED** [review]

**Review by dplumb94 — DISMISSED** [review]

**Review by shannynalayna — DISMISSED** [review]

**Review by dplumb94 — APPROVED** [review]

**Review by dplumb94 — CHANGES_REQUESTED** [review]
"Create new products from a YAML file." is not accurate anymore since we can either use a GDSN XML file, YAML file, or specify properties directly on the CLI.

This comment wasn't addressed

**Comment by KevinAnsleyJohnson** [comment]
> "Create new products from a YAML file." is not accurate anymore since we can either use a GDSN XML file, YAML file, or specify properties directly on the CLI.
> 
> This comment wasn't addressed

Yep - I see that. I just updated the PR.

**Review by dplumb94 — APPROVED** [review]

**Review by vaporos — APPROVED** [review]

**Pull request merged**